### ISSUE TITLE
How to Extend Metric Labelling

### ISSUE THREAD

**Issue by david-rooke** [braedon/prometheus-es-exporter#60]
Hi,
Is it possible to annotate metrics such `es_cluster_health_status` with meaningful labels such as `colour=<green|yellow|red>`? I'm planning some dashboards and this would be very helpful.

**Comment by braedon** [comment]
Hi @david-rooke, thanks for the suggestion.

It's generally better to update values rather than labels when something changes, so how about adding new metrics for each of the colours, which are either 0 or 1, depending on which status a cluster/index has?

That'd allow you to find how many (and if any) clusters/indices are in a given status, which isn't really possible with the current metric. 

e.g. (for a test cluster with 1 index, and 1 node - hence the yellow status)
```
# HELP es_cluster_health_status 
# TYPE es_cluster_health_status gauge
es_cluster_health_status 1.0
# HELP es_cluster_health_status_green 
# TYPE es_cluster_health_status_green gauge
es_cluster_health_status_green 0.0
# HELP es_cluster_health_status_yellow 
# TYPE es_cluster_health_status_yellow gauge
es_cluster_health_status_yellow 1.0
# HELP es_cluster_health_status_red 
# TYPE es_cluster_health_status_red gauge
es_cluster_health_status_red 0.0
...
# HELP es_cluster_health_indices_status 
# TYPE es_cluster_health_indices_status gauge
es_cluster_health_indices_status{index="test"} 1.0
# HELP es_cluster_health_indices_status_green 
# TYPE es_cluster_health_indices_status_green gauge
es_cluster_health_indices_status_green{index="test"} 0.0
# HELP es_cluster_health_indices_status_yellow 
# TYPE es_cluster_health_indices_status_yellow gauge
es_cluster_health_indices_status_yellow{index="test"} 1.0
# HELP es_cluster_health_indices_status_red 
# TYPE es_cluster_health_indices_status_red gauge
es_cluster_health_indices_status_red{index="test"} 0.0
```
The current metric would probably still be best for checking the status of an individual cluster/index though - mapping 0/1/2 to green/yellow/red.

Would that help your use case? If not, if you could give me more detail on what you're trying to do, and I can see if there are other options.

**Comment by david-rooke** [comment]
Hi, Yes that would be helpful. Thank you :)

**Comment by braedon** [comment]
@david-rooke released a version ([0.8.0](https://github.com/braedon/prometheus-es-exporter/releases/tag/0.8.0)) with the extra metrics.
